PodcastGenerator/themes/default/jumbotron.php

```diff
@@ -19,6 +19,16 @@
         }
         ?>
     </div>
+    <div class="search-container" style="margin-bottom: 2rem;">
+        <form action="search.php" class="form-inline">
+            <div class="form-group" style="width: 80%;">
+                <input type="text" class="form-control" placeholder="Search.." name="search" style="width: 98%;">
+            </div>
+            <div class="form-group" style="width: 20%;">
```

Comment:
Is this mobile friendly?

Reply:
Yes, tested with and without freebox as well. Can test here http://lits.co.za/test

Reply:
I ran some tests using that link @kalvinmattheus and it looks nice!